### DISCUSSION TITLE
it so slowly when i use vscode

### DISCUSSION THREAD

**Discussion by shinonomeow** [folke/flash.nvim#191] [Q&A]
```lua
return {
    "folke/flash.nvim",
    event = "VeryLazy",
    vscode = true,
    ---@type Flash.Config
    opts = {},
    -- stylua: ignore
    keys = {
      { "s", mode = { "n", "x", "o" }, function() require("flash").jump() end, desc = "Flash" },
      { "S", mode = { "n", "o", "x" }, function() require("flash").treesitter() end, desc = "Flash Treesitter" },
      { "r", mode = "o", function() require("flash").remote() end, desc = "Remote Flash" },
      { "R", mode = { "o", "x" }, function() require("flash").treesitter_search() end, desc = "Treesitter Search" },
      { "<c-s>", mode = { "c" }, function() require("flash").toggle() end, desc = "Toggle Flash Search" },
    },
  }
```

When I press 's' to search for a word, it usually takes about 2 seconds for the results to appear. Subsequent input also has a similar delay. In contrast, 'S' responds quickly. Is there any way to make 's' in nvim as fast as 'S'? I'm not sure if there's something wrong with my settings.

**Comment by shinonomeow** [comment]
my neovim version is neovim 0.9.1
my system is macos 13.5
VSCode Neovim v0.4.1
Version: 1.80.2 (Universal)

**Comment by shinonomeow** [comment]
Do I have any methods to troubleshoot and identify where the problem is? I'm still a beginner

**Comment by folke** [comment]
It's instant for me. You probably have other plugins enabled that shouldn't in vscode?

**Reply by shinonomeow** [reply]
thank you,I just try to  reinstall VS Code try to reinstall vscode,neovim,flash -neovim,but the delay still persists

**Reply by shinonomeow** [reply]
I found the cause of the problem. vscode-nvim was upgraded from version 0.32 to 0.4 (0/1), which caused this issue. Downgrading the version resolved the problem.

**Comment by shinonomeow** [comment]
i have test on my windows(wsl),its also slowly when i use "s","S" is normal
